packages/core/http/core-http-resources-server-internal/src/http_resources_service.ts

```diff
@@ -23,21 +23,29 @@ import type {
   InternalRenderingServiceSetup,
 } from '@kbn/core-rendering-server-internal';
 import type { RequestHandlerContext } from '@kbn/core-http-request-handler-context-server';
-import {
-  InternalHttpResourcesSetup,
+import type {
   HttpResources,
   HttpResourcesResponseOptions,
   HttpResourcesRenderOptions,
   HttpResourcesRequestHandler,
   HttpResourcesServiceToolkit,
-} from './types';
+} from '@kbn/core-http-resources-server';
+
+import { InternalHttpResourcesSetup } from './types';
+
 import { getApmConfig } from './get_apm_config';
 
+/**
+ * @internal
```

Comment:
These weren't annotated as either public or internal. I went with internal given that the service is mostly internal.